crates/router/src/connector/bluesnap/transformers.rs

```diff
@@ -590,6 +621,14 @@ impl TryFrom<&BluesnapRouterData<&types::PaymentsCompleteAuthorizeRouterData>>
             .parse_value("BluesnapRedirectionResponse")
             .change_context(errors::ConnectorError::ResponseDeserializationFailed)?;
 
+        let transaction_meta_data = match item.router_data.request.metadata.as_ref() {
```

Comment:
Do we need to pass meta data in complete authorize as well?

Reply:
Yes, because transaction call to Bluesnap for 3ds payments happen through complete authorize